config/locales/interview.yml

```diff
@@ -5,13 +5,23 @@ en:
         title: Set up an interview
         interview_preferences:
           title:  Candidate interview preferences
+      edit:
+        title: Change interview details
+        interview_preferences:
+          title:  Candidate interview preferences
       check:
         page_title: Set up Interview
         title: Check and send interview details
         confirm: Send interview details
         cancel: Cancel
+        update:
```

Comment:
I think the update block is a level too deep. However, update is an action, so do we even need this? Line 23 should cover it.